src/components/Discussion/Internal/Comment/Header.js

```diff
@@ -214,43 +216,54 @@ export const Header = ({ t, comment, menu, isExpanded, onToggle }) => {
         }
       })()}
       <div {...styles.center}>
-        <Link displayAuthor={displayAuthor} passHref>
-          <a {...styles.name} {...colorScheme.set('color', 'text')}>
-            {name}
-          </a>
-        </Link>
+        {!published && (
```

Comment:
detail, why not:
`{ published ? (<PublishedStuff/>) : (<UnpublishedStuff/>)}`

Reply:
Thought so, too.

Yet, pattern spread all over these files is usually `foobar && <Something />`, `!foobar && <SomethingElse />` and was inclined to stick with it.

I assume it has some advantages, when …

… condition for truthy changes but not for falsy wouldn't e.g.
```
foobar && buzzfizz && <Something />
!foobar && <SomethingElse />
```

… nested conditions e.g.

```
foobar && !fizzbuzz && <Something />
foobar && fizzbuzz && <SomethingFizzy />
!foobar && <SomethingElse />
```

(instead of)

```
foobar ? (
  fizzbuzz ? (
    <Something />
  ) : (
    <SomethingFizzy />
  )
) : (
  <SomethingElse />
)
```